adminer/include/editing.inc.php

```diff
@@ -290,7 +290,6 @@ function edit_fields($fields, $collations, $type = "TABLE", $foreign_keys = arra
 * @return bool
 */
 function process_fields(&$fields) {
-	ksort($fields);
```

Comment:
Theoretically, browsers could send the fields in a random order. That's why the code was structured this way. I'm not sure if it's worth maintaining this assumption.